modAionImpl/src/org/aion/zero/impl/blockchain/AionImpl.java

```diff
@@ -156,7 +156,7 @@ public long estimateTxNrg(AionTransaction tx, IAionBlock block) {
     @Override
     public AionTxReceipt callConstant(AionTransaction tx, IAionBlock block) {
         if (tx.getSignature() == null) {
-            tx.sign(ECKeyFac.inst().fromPrivate(new byte[32]));
+            tx.sign(ECKeyFac.inst().fromPrivate(new byte[64]));
```

Comment:
We could also use `ECKeyEd25519.SECKEY_BYTES`, or create an abstract method in the `ECKey` class for querying private key length.

Reply:
Implementing this now. Gimme 10 min.

Reply:
Actaully, let's leave it as it is for now. Improvement for later. Don't want to introduce a decorative abstract class with just one function.